### PR TITLE
Implement OBF-GPX customizations (generation/rendering rules)

### PR DESCRIPTION
**Pull request by RZR-UA**

### CODE REVIEW

**Comment by RZR-UA** [comment]
https://github.com/osmandapp/OsmAnd/issues/18866

File: rendering_styles/routes.addon.render.xml [hunk 1]
```diff
@@ -372,6 +377,37 @@
 	</order>
 
 	<text>
+			<!-- OBF-GPX track name/ref/shield -->
+			<switch>
+				<case minzoom="12" tag="route" value="segment" nameTag="" textMinDistance="60" textOnPath="true" textSize="15" textColor="#0078ff" textHaloColor="#ccebf4ff" textHaloRadius="1" textOrder="2"/>
+				<case minzoom="12" tag="route" value="segment" nameTag="ref" textMinDistance="60" textOnPath="false" textSize="15" textColor="#404040" textOrder="2">
+					<!-- blue_hexagon_3_road_shield -->
+					<apply textShield="?gpx_bg?"/>
```
Comment (xmd5a2) [inline rendering_styles/routes.addon.render.xml:385]: Move to line 383

**Review by xmd5a2 — APPROVED** [review]

File: rendering_styles/routes.addon.render.xml (hunk 1 above)
Comment (RZR-UA) [reply rendering_styles/routes.addon.render.xml]: optimized

**Review by xmd5a2 — APPROVED** [review]

File: poi/poi_types.xml [hunk 2]
```diff
@@ -33,14 +33,27 @@
...
 	<poi_additional name="gpx_icon" tag="gpx_icon" type="text"/>
 	<poi_additional name="gpx_bg" tag="gpx_bg"  type="text"/>
 	<poi_additional name="color" tag="colour" type="text"/>
-<!--
-	<poi_additional name="color_int" tag="color_int" type="text" lang="true" order="82"/>
--->
 
-	
+	<!-- OBF-GPX track extensions (color=gpx_color, width already defined, point_groups todo -->
+	<poi_additional name="show_arrows" tag="show_arrows" type="text"/>
```
Comment (ivanPyrohivskyi) [inline poi/poi_types.xml:43]: perhaps better to group to poi_additional_category

Comment (RZR-UA) [reply poi/poi_types.xml]: reorganized with gpx_ prefix

**Pull request merged**